dom.css

```diff
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  width: 100%;
+  height: 100%;
+  margin-left: auto;
+  margin-right: auto;
+}
+/* style header section */
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  font-size: 17px;
+  border: 1px solid black;
+  text-align: end;
+  width: 100%;
+  height: 50px;
+  background: #f1f1f1;
+  padding: 10px;
+}
+.header a {
+  font-size: 20px;
+  font-weight: 600;
+}
+
+/* Style the submit button */
+form.example button {
+  float: left;
+  width: 20%;
+  padding: 10px;
+  background: #2196f3;
+  color: white;
+  font-size: 17px;
+  border: 1px solid black;
+  border-left: none; /* Prevent double borders */
+  cursor: pointer;
+}
+
+/* About section */
+.about {
+  padding: 10px;
+  font-size: 16px;
+  font-weight: 400;
+  border: 1px solid black;
+  width: 80%;
+  height: 150px;
+  background: #f1f1f1;
+  margin-top: 60px;
+}
+/* Experience section */
+#experiences {
+  border: 1px solid black;
+  background: #f1f1f1;
+  padding: 10px;
```

Comment:
Why are we using id selector here. Is there any specific reason for that. We should generally use classes for styling sections so that if that same style is needed somewhere else we can use the same class but id can't serve this purpose because it is unique for every element.